src/pages/docs/agent/troubleshooting/how-to-configure-agent-logs.md

```
@@ -38,14 +38,31 @@ Agent log-level configuration in Testsigma is important for debugging and keepin
-3. After selecting the log-level configuration, **restart** the agent for the changes to take effect. ![choose log level and retsart agent](https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/applications/select_loglevel_restart.gif)
+2. Click on the **Agent Settings** icon in the right navigation bar. Choose the agent log level configuration for CLI and local files from the log level options. 
+   <br>
+
+<div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;">
+  <iframe src="https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/agent/gotoagent-1.mp4" 
+          style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border: 0;" 
+          allowfullscreen>
+  </iframe>
+</div>
+
+  - **ERROR**: Logs only critical issues that cause the agent to fail. Only error messages will appear in the CLI and be saved in the local agent log files.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)